frontend/src/components/common/button/styles.module.scss

```diff
@@ -18,7 +18,7 @@
   cursor: pointer;
 }
 
-.iconButton {
+.button-icon {
```

Comment:
Use camel case

Reply:
<img width="254" alt="image" src="https://user-images.githubusercontent.com/78655088/191295294-194dce59-c0c0-4054-bd16-343d57165a54.png">
I think we have to use hyphen notation because firstly we have this construct in component, and secondly the whole stylesheet is in hyphen notation